includes/discovery/os/eatonups.inc.php

```diff
@@ -1,7 +1,22 @@
 <?php
 if (!$os) {
     // Eaton UPS
-    if (strstr($sysDescr, 'Eaton 5PX')) {
+    if (strstr($sysDescr, 'Eaton 5P')) {
```

Comment:
I think something like this would be easier to read:
```
    if (strstr($sysDescr, 'Eaton 5P') ||
        strstr($sysDescr, 'Eaton 5PX') ||
        strstr($sysDescr, 'Eaton 9130') ||
        strstr($sysDescr, 'Eaton 93E') ||
        strstr($sysDescr, 'Eaton 9PX') ||
        strstr($sysDescr, 'Eaton Evolution') ||
        strstr($sysDescr, 'Eaton EX') ) {

        $os = 'eatonups';
    }
```